src/main/java/io/dockstore/EntryCreatorExample.java

```diff
@@ -160,7 +160,7 @@ Coordinates for core library (see all on [Maven Central]):
         ActionsApi actionsApi = new ActionsApi(client);
         Deposit publishedDeposit = actionsApi.publishDeposit(depositionID);
 
-        String conceptDoiUrl = publishedDeposit.getLinks().get("parent_doi");
```

Comment:
It looks like the production zenodo api has changed. May want to keep an eye on whether automatic DOIs are working still.